src/core/test/datagen/src/main/scala/GenerateDataset.scala

```diff
@@ -1,110 +0,0 @@
-// Copyright (C) Microsoft Corporation. All rights reserved.
-// Licensed under the MIT License. See LICENSE in project root for information.
-
-package com.microsoft.ml.spark
-
-import com.microsoft.ml.spark.ColumnOptions.ColumnOptions
-import com.microsoft.ml.spark.DataOptions.DataOptions
-import org.apache.spark.mllib.random.RandomRDDs
-import org.apache.spark.sql._
-import org.apache.spark.sql.types._
-
-import scala.util.Random
-
-/** Defines methods to generate a random spark DataFrame dataset based on given options. */
-object GenerateDataset {
-
-  /** Generates a random Spark DataFrame given a set of dataset generation constraints.
-    * @param sparkSession The spark session.
-    * @param datasetGenerationConstraints The dataset generation constraints to use.
-    * @param seed The random seed.
-    * @return A randomly generated dataset.
-    */
-  def generateDataset(sparkSession: SparkSession,
```

Comment:
these files are needed to be able to generate datasets with random values based on constraints.  Please don't remove the files:
GenerateDataset.scala
GenerateDataType.scala
DatasetConstraints.scala
DatasetOptions.scala
GenerateRow.scala

I don't see how one could generate random datasets if these are removed.  I had several tests that used these methods, and I planned to add more in the future.